src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtilsUnitTest.java

```diff
@@ -895,6 +934,33 @@ public void testConstructPhaseGroups(final List<VariantContext> calls,
         haplotypeMap.put(vc4, haplotypes4complete);
         tests.add(new Object[]{calls, new HashMap<>(haplotypeMap), 0, 0, 0, 0, 0});
 
+        final Allele refForDel = Allele.create("AA", true);
+        final Allele altDel = Allele.create("A", false);
+
+        final VariantContext delVC = new VariantContextBuilder().chr("20").start(3).stop(4).alleles(Arrays.asList(refForDel, altDel)).make();
+        final VariantContext spannedSnpVC = new VariantContextBuilder().chr("20").start(4).stop(4).alleles(Arrays.asList(ref, Allele.SPAN_DEL, altT)).make();
+
+        final Haplotype spandelHap = new Haplotype("AAAA".getBytes());
```

Comment:
If the ref for the deletion is AA and the remaining haplotype is AAAA, then isn't the deletion not left aligned?  I don't know if that matters at this point in the code, but I don't want to complicate things.

Reply:
Updated this test data with a more informative and easy-to-follow haplotype that avoids this problem.